pkgs/tools/misc/flameshot/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, qtbase, qmake, qttools }:
+
+stdenv.mkDerivation rec {
+  name = "flameshot-${version}";
+  version = "0.5.0";
+
+  nativeBuildInputs = [ qmake qttools ];
+  buildInputs = [ qtbase ];
+
+  qmakeFlags = [
+    # flameshot.pro assumes qmake is being run in a git checkout and uses it
+    # to determine the version being built. Let's replace that.
+    "VERSION=${version}"
+    "PREFIX=/"
+  ];
+  patchPhase = ''
+    sed -i 's/VERSION =/#VERSION =/g' flameshot.pro
+    sed -i 's,USRPATH = /usr/local,USRPATH = /,g' flameshot.pro
```

Comment:
This feels ugly, but not sure if there's a cleaner way. `flameshot.pro` contains hard-coded `USRPATH` that can take one of two values depending on whether `CONFIG` contains `packaging`.

Reply:
For convenience, here's flameshot.pro: https://github.com/lupoDharkael/flameshot/blob/v0.5.0/flameshot.pro